### PR TITLE
2021-05-17-visual_document_NER_SROIE0518_en

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

**Review by kolia1985 — CHANGES_REQUESTED** [review]

File: docs/_posts/ChicoQ/2021-05-17-visual_document_NER_SROIE0518_en.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+layout: model
+title: Visual Document NER with SROIE
+author: John Snow Labs
+name: visual_document_NER_SROIE0518
+date: 2021-05-17
+tags: [en, licensed]
+task: Named Entity Recognition
+language: en
+edition: Spark NLP 2.5.5
```
[the remark this answers is not in the document]
Comment (ChicoQ) [reply docs/_posts/ChicoQ/2021-05-17-visual_document_NER_SROIE0518_en.md:10]: I'll change it.

File: docs/_posts/ChicoQ/2021-05-17-visual_document_NER_SROIE0518_en.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+```
+
+{:.model-param}
+## Model Information
+
+{:.table-model}
+|---|---|
+|Model Name:|visual_document_NER_SROIE0518|
+|Type:|ocr|
+|Compatibility:|Spark NLP 2.5.5+|
```
[the remark this answers is not in the document]
Comment (ChicoQ) [reply docs/_posts/ChicoQ/2021-05-17-visual_document_NER_SROIE0518_en.md:85]: will change it.

File: docs/_posts/ChicoQ/2021-05-17-visual_document_NER_SROIE0518_en.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      .pretrained(testSparkModel, "en", "public/ocr/models")
+      .setInputCol("hocr")
+```
+</div>
+
+## Results
+
+```bash
++----------+----------+---------+
+|word      |token     |label    |
```
[the remark this answers is not in the document]
Comment (ChicoQ) [reply docs/_posts/ChicoQ/2021-05-17-visual_document_NER_SROIE0518_en.md:56]: Is that correct?

+--------------------------------------------------------------------+------------+------------+---------+
|entities                                                            |word        |token       |label    |
+--------------------------------------------------------------------+------------+------------+---------+
|[entity, 0, 0, O, [word -> 0£0, token -> 0£0], []]                  |0£0         |0£0         |O        |
|[entity, 0, 0, O, [word -> 0£0, token -> 0£0], []]                  |0£0         |0£0         |O        |
|[entity, 0, 0, O, [word -> 0£0, token -> 0£0], []]                  |0£0         |0£0         |O        |
|[entity, 0, 0, O, [word -> 2-2,, token -> 2], []]                   |2-2,        |2           |O        |
|[entity, 0, 0, O, [word -> 2-2,, token -> -], []]                   |2-2,        |-           |O        |
|[entity, 0, 0, O, [word -> 2-2,, token -> 2], []]                   |2-2,        |2           |O        |
|[entity, 0, 0, O, [word -> 2-2,, token -> ,], []]                   |2-2,        |,           |O        |
|[entity, 0, 0, B-COMPANY, [word -> AEON, token -> aeon], []]        |AEON        |aeon        |B-COMPANY|
|[entity, 0, 0, B-COMPANY, [word -> AEON, token -> aeon], []]        |AEON        |aeon        |B-COMPANY|
|[entity, 0, 0, B-COMPANY, [word -> CO., token -> co], []]           |CO.         |co          |B-COMPANY|
|[entity, 0, 0, B-COMPANY, [word -> CO., token -> .], []]            |CO.         |.           |B-COMPANY|

root
 |-- image: struct (nullable = true)
 |    |-- origin: string (nullable = true)
 |    |-- height: integer (nullable = false)
 |    |-- width: integer (nullable = false)
 |    |-- nChannels: integer (nullable = false)
 |    |-- mode: integer (nullable = false)
 |    |-- resolution: integer (nullable = false)
 |    |-- data: binary (nullable = true)
 |-- path: string (nullable = true)
 |-- modificationTime: timestamp (nullable = true)
 |-- length: long (nullable = true)
 |-- pagenum: integer (nullable = true)
 |-- corrected_image: struct (nullable = true)
 |    |-- origin: string (nullable = true)
 |    |-- height: integer (nullable = false)
 |    |-- width: integer (nullable = false)
 |    |-- nChannels: integer (nullable = false)
 |    |-- mode: integer (nullable = false)
 |    |-- resolution: integer (nullable = false)
 |    |-- data: binary (nullable = true)
 |-- exception: string (nullable = true)
 |-- hocr: string (nullable = true)
 |-- entities: struct (nullable = true)
 |    |-- annotatorType: string (nullable = true)
 |    |-- begin: integer (nullable = false)
 |    |-- end: integer (nullable = false)
 |    |-- result: string (nullable = true)
 |    |-- metadata: map (nullable = true)
 |    |    |-- key: string
 |    |    |-- value: string (valueContainsNull = true)
 |    |-- embeddings: array (nullable = true)
 |    |    |-- element: float (containsNull = false)
 |-- word: string (nullable = true)
 |-- token: string (nullable = true)
 |-- label: string (nullable = true)

**Comment by ChicoQ** [comment]
as pavel's suggestion, shall we close this PR?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
pavel 
Hello, @here. If you want to change Spark NLP 2.5.5 to Spark NLP 3.x, you have to close this pull request without merging and upload this model again. Otherwise, the model file on S3 and auxdata.johnsnowlabs.com/clinical/ocr/metadata.json file will contain 2.5.5. (edited)

**Pull request closed without merging**